rescript/replace_taxonomy.py

```diff
@@ -0,0 +1,45 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2021, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from qiime2.plugin import Metadata
+import pandas as pd
+import re
+
+
+def make_substitutions_dict(taxonomy_replacement_map):
+    if taxonomy_replacement_map.column_count < 1:
+        raise ValueError("Metadata file should only contain one column of "
```

Comment:
can skip this validation if you accept a metadata column instead of metadata

Reply:
Super duper cool! Done.